src/lib/components/LoginModal.svelte

```diff
@@ -31,17 +28,15 @@
 		<p class="px-2 text-sm text-gray-500">
 			Your conversations will be shared with model authors unless you disable it from your settings.
 		</p>
-		<form action="{base}/settings" use:enhance method="POST">
-			<input type="hidden" name="ethicsModalAccepted" value={true} />
-			{#each Object.entries(settings) as [key, val]}
-				<input type="hidden" name={key} value={val} />
-			{/each}
+		<form action="{base}/login" use:enhance method="POST">
+			<input type="hidden" name="callbackUrl" value={base} />
```

Comment:
I think we still want to allow users who don't define `HF_CLIENT_ID` to be able to run the app locally, so for them, the form should be unchanged.

Regarding `<input type="hidden" name="callbackUrl" value={base} />`: it's currently not used by the backend.